### PR TITLE
Thirsty Hakkar

### LINKED ISSUE #447: [SUGGESTION] | Give Hakkar vanilla Aztec/bloodthirsty doctrine decisions
**Describe your suggestion in full detail below:**

Title

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Adapted vanilla `Sacrifice` decision for Hakkar Cult.
- Adapted gaining `Haemophiliac`, `Haemophant`, `Haemoarch` traits and `Bloodthirsty` bloodline for Hakkar Cult.
- Adapted vanilla `Flower War` CB for Hakkar Cult.
- Adapted vanilla `Perform a Mass Sacrifice` decision for Hakkar Cult.

#### Developer changelog:
- Moved unused blood rituals decision and event to coder resources
- Deleted unused sunset invasion events
- Commented `vassal_generic_sacrifice_decision`
- Close #447

#### How to test:
- Check decisions, events and its localization

### CODE REVIEW

**Comment by zumbak04** [comment]
**Hold Blood Rituals** is here, though it does the same thing as **Sacrifice** and has the same meaning as **Perform a Mass Sacrifice**.

<details>
<summary>Click to expand</summary>

image

</details>

I would like to see more cannibals among the Hakkari: some kind of adaptation of `WCLOA.110` where you and your court become cannibals.

**Comment by zumbak04** [comment]
Clicking **Sacrifice** is a real pain, so I suggest adding a decision to sacrifice all your prisoners.

**Comment by arithon** [comment]
Can we get buttons to put all prisoners into house arrest/oubliette as well pls?

**Comment by zumbak04** [comment]
> Can we get buttons to put all prisoners into house arrest/oubliette as well pls?

Soon™

**Comment by zumbak04** [comment]
Could you replace all `has_religion_feature = religion_ritual_sacrifice` by `religion_have_ritual_sacrifice_trigger`? Distracting attention.

**Comment by zumbak04** [comment]
Thinking of `sunset_invasion.txt`. These left events are pretty useless... and I have no idea why Paradox added them. Maybe remove?

**Comment by zumbak04** [comment]
> Maybe remove?

Or unite with the Old Gods' sacrifice somehow. There is a lot of religions that would like to kill someone.

**Comment by MemeWizard** [comment]
> Hold Blood Rituals is here, though it does the same thing as Sacrifice and has the same meaning as Perform a Mass Sacrifice.

> Clicking Sacrifice is a real pain, so I suggest adding a decision to sacrifice all your prisoners.

So, you want to remove "Hold Blood Rituals", a decision to sacrifice all your prisoners???


> Can we get buttons to put all prisoners into house arrest/oubliette as well pls?

You should check [this](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/issues/new?labels=%3Aquestion%3A+suggestion+%3Aquestion%3A&template=suggestion-request.md&title=Suggestion+Title) thing

> Could you replace all has_religion_feature = religion_ritual_sacrifice by religion_have_ritual_sacrifice_trigger?

Ok, will try

> Thinking of sunset_invasion.txt. These left events are pretty useless... and I have no idea why Paradox added them. Maybe remove?

Didn't get what you mean. What events?
Sacrificing courtiers when you siege holding?

> I would like to see more cannibals among the Hakkari: some kind of adaptation of WCLOA.110 where you and your court become cannibals. I suggest doing in Sacrifice because AI rarely does Perform a Mass Sacrifice.

Will see what can I do with that.

**Comment by zumbak04** [comment]
> > Clicking Sacrifice is a real pain, so I suggest adding a decision to sacrifice all your prisoners.
> 
> So, you want to remove "Hold Blood Rituals", a decision to sacrifice all your prisoners???

It has other functions besides prisoner sacrificing. Also, I would call the decision different... but **Perform a Mass Sacrifice** is taken, **SAD**.
I would rename **Perform a Mass Sacrifice** as **Hold Blood Rituals** and name the new sacrifice 
decision **Perform a Mass Sacrifice**.

> Didn't get what you mean. What events?
> Sacrificing courtiers when you siege holding?

Yep.

File: localisation/00_wc_vanilla_replacements.csv [hunk 1]
```diff
@@ -153,4 +153,33 @@ HF_222_DESC;My servants apparently lost track of [child_of_destiny.GetFirstName]
 HF_222_OPTA;[child_of_destiny.GetFirstName], oh how you scared me...! And how do you know Kalimag?;;[child_of_destiny.GetFirstName], oh wie habt Ihr mich erschreckt...! Und woher kennt Ihr Kalimag?;;;;;;;;;;;x
 child_of_mithridates;Child of Shadra;;Kind von Shadra;;;;;;;;;;;x
 child_of_mithridates_desc;This character resists poison to such a degree that people claim [GetHerHim] a child of Shadra.;;Dieser Charakter widersteht Gift in einem solchen Ma�e, dass die Leute behaupten, [GetSheHe] sei ein Kind von Shadra.;;;;;;;;;;;x
-bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of the ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
\ No newline at end of file
+bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of the ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_wc_vanilla_replacements.csv]:
```suggestion
bloodline_desc_child_of_destiny;A [child_of_destiny.GetManWoman] of true legend. [child_of_destiny.GetHerHisCap] exploits mimicked those of ancient heroes!;;Ein wahres [child_of_destiny.GetManWoman] der Legenden. [child_of_destiny.GetSheHeCap] eifert den Heldentaten der alten Helden nach!;;;;;;;;;;;x
```

**Comment by MemeWizard** [comment]
> Clicking Sacrifice is a real pain, so I suggest adding a decision to sacrifice all your prisoners.

Added in 691d60f2

> Hold Blood Rituals is here, though it does the same thing as Sacrifice and has the same meaning as Perform a Mass Sacrifice.

Disabled it in 88be3569

>I would like to see more cannibals among the Hakkari: some kind of adaptation of WCLOA.110 where you and your court become cannibals. I suggest doing in Sacrifice because AI rarely does Perform a Mass Sacrifice.

Adapted in 53d664a4

File: localisation/00_wc_vanilla_replacements.csv (hunk 1 above)
Comment (MemeWizard) [reply localisation/00_wc_vanilla_replacements.csv]: It is not part of this PR
@zumbak04. @arithon, can you apply this suggestion it in the next localization branch, please?

Comment (arithon) [reply localisation/00_wc_vanilla_replacements.csv]: he mentioned it in a couple PRs recently... did anyone change it anywhere?

Comment (zumbak04) [reply localisation/00_wc_vanilla_replacements.csv]: @DukeJeans @MemeWizard @arithon I already changed it on the bug crusade.

**Comment by MemeWizard** [comment]
> Or unite with the Old Gods' sacrifice somehow. There is a lot of religions that would like to kill someone.

Did it in 1a87e716

**Comment by zumbak04** [comment]
@MemeWizard what about Hakkar event picture instead of `GFX_evt_pagan_blood_sacrifice`?

**Comment by MemeWizard** [comment]
> what about Hakkar event picture instead of `GFX_evt_pagan_blood_sacrifice`?

Will add this to my ToDo list

**Pull request merged**